.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2
+
+            - name: Set up Node.js
+              uses: actions/setup-node@v2
+              with:
+                  node-version: "22"
+
+            - name: Install dependencies
+              run: npm install
+
+            - name: Run lint
+              run: npm run lint
+
+            - name: Run build
+              run: npm run build
+
+            # - name: Run tests
```

Comment:
we need to decide on a testing framework

Reply:
is this tracked here https://satellite-scheduler.atlassian.net/browse/SS-8 ?

Reply:
lol i forgot about that ticket, but yea whenever we decide on a framework (https://satellite-scheduler.atlassian.net/browse/SS-25), we should then just uncomment this section and wrap up SS-8